src/chart/line/LineView.ts

```diff
@@ -644,7 +644,7 @@ class LineView extends ChartView {
 
         const lineGroup = this._lineGroup;
 
-        const hasAnimation = seriesModel.get('animation');
+        const hasAnimation = !ecModel.ssr && seriesModel.isAnimationEnabled();
```

Comment:
`seriesModel.isAnimationEnabled` should have considerd `ecModel.ssr`

Reply:
But it requires `env.node`. https://github.com/apache/echarts/blob/master/src/model/Series.ts#L469

Reply:
Understand, we tried to fix the issue by disabling the animation in SSR mode.